test/helpers.bash

```diff
@@ -555,25 +552,23 @@ function is_cgroup_v2() {
 
 function create_runtime_with_allowed_annotation() {
     local NAME="$1"
-    local ANNOTATION="$2"
+    local ANNOTATION="${2:-}"
     cat <<EOF >"$CRIO_CONFIG_DIR/01-$NAME.conf"
 [crio.runtime]
 default_runtime = "$NAME"
 [crio.runtime.runtimes.$NAME]
-runtime_path = "$RUNTIME_BINARY_PATH"
-runtime_root = "$RUNTIME_ROOT"
-runtime_type = "$RUNTIME_TYPE"
-allowed_annotations = ["$ANNOTATION"]
+runtime_path = "${RUNTIME_BINARY_PATH:-}"
+runtime_root = "${RUNTIME_ROOT:-}"
+runtime_type = "${RUNTIME_TYPE:-}"
+allowed_annotations = ["${ANNOTATION:-}"]
```

Comment:
Setting some parameters to empty values seem questionable (even if it was done the same way before).

So maybe do it conditionally instead?

Also, it seems weird that we are adding runtime without adding the path to its binary.

Reply:
Yes, now that I actually tried to read the setup code paths, it looks to me as if all of those are always set.